docs/deployment/rds-s3/README.md

```diff
@@ -4,7 +4,28 @@
 
 This Kustomize Manifest can be used to deploy Kubeflow Pipelines and Katib with RDS and S3.
 
-Follow the [install](#install) steps below to configure and deploy the Kustomize manifest.
+### RDS
+
+[Amazon Relational Database Service (RDS)](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html) is a managed relational database service that facilitates several database management tasks such as database scaling, database backups, database software patching, OS patching, and more.
+
+In the [default kubeflow installation](../../../../example/kustomization.yaml), the [KFP](../../../../apps/katib/upstream/components/mysql/mysql.yaml) and [Katib](../../../../apps/pipeline/upstream/third-party/mysql/base/mysql-deployment.yaml) components both use their own MySQL pod to persist KFP data (such as experiments, pipelines, jobs, etc.) and Katib experiment observation logs, respectively. 
```

Comment:
persist metadata? since both pipeline and katib use it

Kubeflow Pipeline (KFP) -> should do this when we use a short form

Reply:
../../../../example/kustomization.yaml, ../../../../apps/katib/upstream/components/mysql/mysql.yaml broken links. same for other places